File: Projekt Aplikacje/ClientApp/src/app/models/dataGroup.ts
import { DataModel } from 'src/app/models/dataModel';

interface IDataGroup {
  id: number;
  name: string;
  unit: string;
  chartType: string;
  datas: DataModel[];
}

export class DataGroup implements IDataGroup {
  id: number;
  name: string;
  unit: string;
  chartType: string;
  datas: DataModel[];

  public constructor(init?: Partial<DataGroup>) {
    Object.assign(this, init);
  }
}

File: Projekt Aplikacje/Models/DataGroup.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using Todo.Models;

namespace Projekt_Aplikacje.Models
{
    public class DataGroup
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public string Unit { get; set; }

        public string ChartType { get; set; } = "bar";

        public ICollection<DataModel> Datas { get; set; }
    }
}
